mmseg/datasets/transforms/transforms.py

```diff
@@ -1226,3 +1226,129 @@ def __repr__(self):
         repr_str += f'edge_width={self.edge_width}, '
         repr_str += f'ignore_index={self.ignore_index})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class RandGaussianNoise(BaseTransform):
+    """Add random Gaussian noise to image.
+
+    Modified from https://docs.monai.io/en/stable/_modules/monai/transforms/intensity/array.html#RandGaussianNoise
+
+    Required Keys:
+
+    - img
+
+    Modified Keys:
+
+    - img
+
+    Args:
+        prob (float): Probability to add Gaussian noise. Default to 0.1.
+        mean (float): Mean or “centre” of the distribution. Default to 0.0.
+        std (float): Standard deviation (spread) of distribution. Default to 0.1.
+    """
+
+    def __init__(self, prob: float = 0.1, mean: float = 0.0, std: float = 0.1) -> None:
+        super().__init__()
+        self.prob = prob
+        self.mean = mean
+        self.std = std
+        self.R = np.random.RandomState()
+
+    def transform(self, results: Dict) -> Dict:
+        """Call function to add random Gaussian noise to image.
+
+        Args:
+            results (dict): Result dict.
+
+        Returns:
+            dict: Result dict with random Gaussian noise.
+        """
+        do_transform = self.R.rand() < self.prob
```

Comment:
`self.R` is defined in [def set_random_state](https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/transform.py#L148-L180).

Inherit Order: `RandGaussianNoise` -> `RandomizableTransform` -> `Randomizable`.